website/src/components/EventSlider/index.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import Slider from "react-slick";
+import { useTranslation } from "react-i18next";
+import { Typography, Link, Box } from "@material-ui/core";
+import useStyles from "./style";
+import events from '../../resources/events.json';
+
+const EventSlider: React.FC = () => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  const NextArrow = (props: any) => {
+    const { className, onClick } = props;
+    return (
+      <div
+        className={`${className} ${classes.slickButtons}`}
+        onClick={onClick}
+      >
+        <img
+          src="../Images/svg/right_arrow.svg"
+          alt={t("home.adaptorsTestimonials.nextArrowAlt")}
+        />
+      </div>
+    );
+  };
+
+  const PrevArrow = (props: any) => {
+    const { className, onClick } = props;
+    return (
+      <div
+        className={`${className} ${classes.slickButtons}`}
+        onClick={onClick}
+      >
+        <img
+          src="../Images/svg/left_arrow.svg"
+          alt={t("home.adaptorsTestimonials.previousArrowAlt")}
+        />
+      </div>
+    );
+  };
+
+  const settings = {
+    infinite: true,
+    autoplay: false,
+    speed: 500,
+    slidesToShow: 3,
+    slidesToScroll: 1,
+    prevArrow: <PrevArrow />,
+    nextArrow: <NextArrow />,
+    className: `${classes.slidewrap} center`,
+    responsive: [
+      {
+        breakpoint: 1024,
+        settings: {
+          slidesToShow: 2,
+          swipeToSlide: true,
+        },
+      },
+      {
+        breakpoint: 768,
+        settings: {
+          slidesToShow: 1,
+          infinite: false,
+          arrows: false,
+        },
+      },
+    ],
+  };
+
+  function FetchDate (date: any) {
```

Comment:
Let's use camelcase convention of naming.

Reply:
`FetchDate` is used as a component, thats why pascalcase was used.